deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -355,6 +360,19 @@ def get_base_images(request):
     })
 
 
+def get_base_image_events(request, image_id):
```

Comment:
This should be a method of a view class. Like `class EnvCapacityAdvCreateView(View):`

Reply:
let me double check, thanks ^^

Reply:
we may do this in another pr, together with the comment for [cluster_urls.py](https://github.com/pinterest/teletraan/pull/1114/files#diff-a01df602769c1958b47b072b73ef91ded8b6fb4f84e568ecc95b7a82c5f75a8f)